test/lib/linter/UI5YamlLinter.ts

```diff
@@ -0,0 +1,52 @@
+import test from "ava";
+import UI5YamlLinter from "../../../src/linter/yaml/UI5YamlLinter.js";
+import LinterContext from "../../../src/linter/LinterContext.js";
+
+test("Test UI5YamlLinter report (parsing and analyzing)", async (t) => {
+	/* Mock resource content of ui5.yaml file,
+	(formatted as used in src/detectors/typeChecker/index.ts - #analyzeFiles()),
+	(contains relevant 'framework' property and 'libraries' sub-property),
+	(contains only deprecated libraries) */
+	const resourceContent =
+`specVersion: '3.0'
+metadata:
+  name: ava-test-ui5yamllinter
+type: application
+framework:
+  name: OpenUI5
+  version: "1.121.0"
+  libraries:
+    - name: sap.ca.scfld.md
+    - name: sap.ca.ui
+    - name: sap.fe.common`;
+
+	const resourcePath = "/ui5.js"; // '.js' due to renaming in src/detectors/typeChecker/index.ts - #analyzeFiles()
+	const projectPath = "test.ui5yamllinter";
+	const context = new LinterContext({rootDir: projectPath});
+
+	// Create UI5YamlLinter instance with resource content
+	const linter = new UI5YamlLinter(resourcePath, resourceContent, context);
+	// Run UI5YamlLinter report
+	await linter.lint();
+
+	const messages = context.getLintingMessages("/ui5.js");
+
+	// Test returned messages
+	t.is(messages.length, 3, "Detection of 3 deprecated libraries expected");
+
+	// Test each message
+	t.is(messages[0].ruleId, "ui5-linter-no-deprecated-api", `RuleId is correct`);
+	t.is(messages[0].message, `Use of deprecated library 'sap.ca.scfld.md'`, `Message is correct`);
```

Comment:
```suggestion
	t.is(messages[0].message, `Use of deprecated library 'sap.ca.scfld'`, `Message is correct`);
```

Reply:
I don't understand :/ 
This library is called `sap.ca.scfld.md`, right? It's the same name as defined in our utils constant and in the original BLI CPOUI5FOUNDATION-807.

Reply:
Sorry. You're right. I was wrong.